ncm-spma/src/main/perl/spma/yumdnf.pm

```diff
@@ -0,0 +1,28 @@
+use strict;
+use warnings;
+
+package NCM::Component::spma::yumdnf;
+
+use NCM::Component::spma::yum;
+push our @ISA , qw(NCM::Component::spma::yum);
+
+use constant YUM_CONF_FILE => "/etc/dnf/dnf.conf";
+
+use constant REPOQUERY_FORMAT => qw(--nevra);
+
+
+# Completes any pending transactions
+sub _do_complete_transaction
+{
+    my ($self) = @_;
+
+    # Could be implmented using e.g. dnf remove $(dnf repoquery --duplicated --latest-limit=-1 -q)
+    #   not sure it does what you think it does
+    $self->verbose("Skipping complete_transaction, no DNF equivalent implemented");
```

Comment:
This doesn't feel like a message for users but instead programmers so should be debug() not verbose()? Or even disabled all together?

Reply:
I've made this a `debug 1` message.